package.json

```diff
@@ -45,23 +45,24 @@
     "dayjs": "1.10.7",
     "dotenv-cli": "4.1.1",
     "form-data": "4.0.0",
-    "formik": "2.2.9",
     "formidable": "2.0.1",
+    "formik": "2.2.9",
     "iron-session": "6.0.5",
     "lru-cache": "6.0.0",
     "next": "12.0.9",
     "next-connect": "0.11.0",
-    "nextjs-progressbar": "0.0.13",
     "next-seo": "4.29.0",
+    "nextjs-progressbar": "0.0.13",
     "pino": "7.6.4",
     "pino-pretty": "7.5.0",
     "prop-types": "15.8.1",
     "react": "17.0.2",
     "react-bootstrap": "2.1.1",
     "react-dom": "17.0.2",
     "react-paginate": "8.1.0",
-    "safe-json-stringify": "1.2.0",
     "react-toastify": "8.1.0",
+    "react-youtube": "^7.14.0",
```

Comment:
Please use just the version number instead of `^` to avoid breaking updates.

Reply:
Ah yes, I didn't add it manually; just with npm i